zcash_client_sqlite/src/wallet/init/migrations/sapling_memo_consistency.rs

```diff
@@ -108,7 +108,8 @@ impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
                     ))
                 })?;
 
-            let decrypted_outputs = decrypt_transaction(&self.params, block_height, &tx, &ufvks);
+            let decrypted_outputs =
+                decrypt_transaction(&self.params, Some(block_height), None, &tx, &ufvks);
```

Comment:
This decrypts the transaction according to its expiry height if it was not mined, which seems fine.